### DISCUSSION TITLE
PacketShare.io

### DISCUSSION THREAD

**Discussion by af0rte** [engageub/InternetIncome#143] [General]
I'm trying to use packetsahre.io and I noticed that there is no configuration in its scripts.
I added the following to the files:

properties.conf
  # Set your HoneyGain email and password
  PACKETSHARE_EMAIL=email
  PACKETSHARE_PASSWORD=password

internetincome.sh

  # Starting PacketShare container
  if [[ $PACKETSHARE_EMAIL && $PACKETSHARE_PASSWORD ]]; then
    echo -e "${GREEN}Starting PacketShare container..${NOCOLOUR}"
    if [ "$container_pulled" = false ]; then
          sudo docker pull packetshare/packetshare:latest
    fi
    if CONTAINER_ID=$(sudo docker run -d --name packetshare$UNIQUE_ID$i --restart=always $LOGS_PARAM $NETWORK_TUN packetshare/packetshare:latest -email=$PACKETSHARE_EMAIL -password=$PACKETSHARE_PASSWORD -accept-tos); then
      echo "$CONTAINER_ID" | tee -a $containers_file
      echo "PacketShare$UNIQUE_ID$i" | tee -a $container_names_file
    else
      echo -e "${RED}Failed to start container for PacketShare..${NOCOLOUR}"
    fi
  else
    if [ "$container_pulled" = false ]; then
      echo -e "${RED}PacketShareEmail or Password is not configured. Ignoring PacketShare..${NOCOLOUR}"
    fi
  fi

But when i use this comand " sudo bash internetIncome.sh --delete ", it doesn't stop packetshare containers, why?

**Comment by engageub** [comment]
Hello,
The name mentioned in docker run command and containers_file are different. That's why its not getting deleted.
`packetshare$UNIQUE_ID$i` and `PacketShare$UNIQUE_ID$i` are not same as the code is case-sensitive.
Packetshare will be added to the script soon. 

Thank you

**Comment by engageub** [comment]
PacketShare has been added to main branch now.

https://github.com/engageub/InternetIncome

Thank you
